### PR TITLE
Input: fix #5322

### LINKED ISSUE #5322: ant-input-preSuffix-wrapper 这个驼峰命名是否符合规范
#### Environment(required)

- antd@2

#### What did you do? Please provide steps to re-produce your problem.

https://github.com/ant-design/ant-design/blob/070ca82bc2461b68456ab97bb80a0fda4315aff3/components/input/style/mixin.less#L191

    .@{inputClass}-preSuffix-wrapper {
        ...
    }

#### What do you expected?

按之前看到的应该是 `pre-suffix`？我在做样式覆盖，所以这个类名影响到我了。

### PR DESCRIPTION
**Pull request by keqingrong**

### CODE REVIEW

**Comment by afc163** [comment]
preSuffix => pre-suffix

**Comment by benjycui** [comment]
Your snapshot is a little strange. Try `rm -rf node_modules && npm i && npm test -- -u`

File: components/form/__tests__/__snapshots__/demo.test.js.snap [hunk 1]
```diff
@@ -1034,7 +1034,7 @@ exports[`test renders ./components/form/demo/register.md correctly 1`] = `
               class="ant-checkbox-inner" />
           </span>
           <span>
-            I have read the 
+            I have read the
```
Comment (afc163) [inline components/form/__tests__/__snapshots__/demo.test.js.snap]: This breaks ci. It shouldn't be modified.

**Comment by afc163** [comment]
rebase master

**Comment by afc163** [comment]
#5322

**Pull request closed without merging**